test/dev:js/demo.js

```diff
@@ -95,6 +95,21 @@ class Demo extends React.Component {
             buttonStyle={{ borderRadius: '5px 0 0 5px' }}
             dropdownStyle={{ width: '300px' }}
           />
+          <p>Disable initial country guess</p>
```

Comment:
Please remove this test! Just add `disableInitialCountryGuess` here: https://github.com/bl00mber/react-phone-input-2/blob/d8095bf90014582b81ee9c663b2d64e99d739f5f/test/dev:js/demo.js#L177 with value '1300' (and remove invalid country from that test)

Reply:
Not sure if I understand this comment. My idea was to have a specific example of using this new prop `disableInitialCountryGuess`. For now I just removed the example that I added.

If I just add this new prop `disableInitialCountryGuess` to this demo in line 177 I will be changing the `Custom Masks & Area Codes`:
![image](https://user-images.githubusercontent.com/347030/79741980-6ff93e00-8302-11ea-8777-4a4420120824.png)
but to demonstrate the new prop usage we need an example where we pass an initial value.